style.css

```diff
@@ -455,13 +425,184 @@ form {
   min-height: 170px;
 }
 
+.contact-form textarea:focus {
+  outline: none !important;
+  border: 4px solid #000000a6;
+}
+
 .form button {
   margin: 0;
   padding: 12px 16px;
   width: auto;
   border-radius: 8px;
 }
 
+.form button:hover {
+  border: 1px solid #fff;
+  color: #fff;
+  background-color: #1e2bb9;
+}
+
+@media only screen and (max-width: 576px) {
+  /* ######### Menu Pop-up ##########  */
+
+  .navLinks.show {
+    display: flex;
+    flex-direction: row-reverse;
+    justify-content: space-between;
+    position: absolute;
+    left: 0;
+    top: 0;
+    height: 100vh;
+    width: 100vw;
+    padding-left: 30px;
+    padding-right: 42px;
+    padding-top: 26px;
+    background: rgba(97, 113, 255, 0.92);
+    text-align: center;
+    z-index: 2;
+    transition: 0.5s;
+    color: #fff;
+  }
+
+  .navLinks ul {
+    margin-top: 4rem;
+  }
+
+  .navLinks li {
+    margin: 20px 0;
+    font-weight: 600;
+    font-size: 32px;
+    line-height: 44px;
+    text-align: left;
+  }
+}
+
+/* ######### Project Pop-up ##########  */
+
+.project-pop {
+  position: fixed;
+  z-index: 10;
+  top: 0;
+  bottom: 0;
+  width: 100%;
+  max-height: 100vh;
+  background: rgba(0, 0, 0, 0.3);
+  backdrop-filter: blur(3px);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.project-pop-container {
+  position: absolute;
+  z-index: 20;
+  width: 87%;
+  padding: 0 5%;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+}
+
+.project-pop-container img {
+  width: 100%;
+  height: 220px;
+  margin-top: 20px;
+}
```

Comment:
- [`OPTIONAL`] Great job implementing the pop-up menu as specified in the project requirement! 👏 Your design looks great and matches that of the `Figma`. However, I would suggest you give your `image` a `height` of `100%` that way it will scale to the full `height` of the model on different screen sizes. To have a better understanding of what I mean you can resize your screen to `500px` and set the `height` to `100%` and see the change it adds to your beautiful design. Great job once again. 🤗🍀